### PR TITLE
Melting Love display status effect and tweaks [DONE]

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
Changes melting loves status effect  into a sorted display icon. New 10x10 icon provided by anonmare.
The old icon we may salvage for a item since its a nice looking gloop lump.
image

I also had to rewrite some of melting loves code. While testing i noticed that if the user dies while in front of melting love they are converted into a small and big slime. So i added a consideration in slime conversion for the status effect.

I also changed the gib in melting love to not drop bodyparts or organs. This is mostly to try and reduce the amount of entities it creates. Also it turns out there is unique gib sprites for when a person is gibbed(no_bodyparts).

Additionally due to the fact that we have other transformative curses i put in a consideration for melting loves blessing for naked nest and parasite tree.
Melting loves transform will override the transformation from naked nest and parasite tree. So when you go insane with parasite tree due to it breaching you will not spawn a sapling but instead a slime.

## Why It's Good For The Game
Makes melting loves status effect fit with other modular status effects.

## Changelog
:cl:
add: melting love blessing
tweak: melting love
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm [hunk 1]
```diff
@@ -123,21 +117,24 @@
...
 	. = ..()
-	if(prob(33) && user == gifted_human)
+	if(prob(33) && user.has_status_effect(STATUS_EFFECT_MELTYLOVE))
 		datum_reference.qliphoth_change(1)
 	return
 
 /mob/living/simple_animal/hostile/abnormality/melting_love/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
 	. = ..()
-	if(prob(50))
+	if(prob(50) && user.has_status_effect(STATUS_EFFECT_MELTYLOVE))
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm:137]: change not documented in the PR's changelog, nor in the records of melting love

Comment (InsightfulParasite) [reply code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm:137]: Oh i thought it needed the user effect his is a accident.

File: code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm [hunk 2]
```diff
@@ -156,65 +153,43 @@
 	base_pixel_x = -32
 	desc = "A pink hunched creature with long arms, there are also visible bones coming from insides of the slime."
 	if(istype(gifted_human))
-		to_chat(gifted_human, span_userdanger("You feel like you are about to burst !"))
 		gifted_human.emote("scream")
-		if(KillGifted())
-			UnregisterSignal(gifted_human, COMSIG_LIVING_DEATH)
-			UnregisterSignal(gifted_human, COMSIG_WORK_COMPLETED)
+		gifted_human.death()
```
Comment (Coxswain-Navigator) [inline code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm:157]: This removes the ability to survive losing the gift from ML which was introduced in one of the more recent reworks.

**Review by Kitsunemitsu — APPROVED** [review]
Code looks good to me, mostly just a refactor.

**Pull request merged**